docs/services/motion/_index.md

```diff
@@ -0,0 +1,466 @@
+
+- `worldState` ([WorldState](https://pkg.go.dev/go.viam.com/rdk/referenceframe#WorldState)): Data structure specifying information about the world around the robot.
+  Used to augment the motion solving process.
+  `worldState` includes obstacles and transforms:
+  - **Obstacles**: Geometries located at a pose relative to some frame.
+    When solving a motion plan with movable frames that contain inherent geometries, the solved path is constrained such that none of those inherent geometries intersect with the obstacles.
+    Important considerations:
+    - If a motion begins with a component already in collision with an obstacle, collisions between that specific component and that obstacle will not be checked.
+    - The Motion Service assumes that obstacles with mobile parents move along with their parents while solving.
+      This ensures that obstacles that are temporarily attached to moving components do not cause collisions during movement.
+    - Geometries are "part of" their frame, rather than at the distal end of the frame.
+      Their poses are relative to the *origin* of the specified frame.
+      A geometry associated with the frame of an arm with a pose of {X: 0, Y: 0, Z: -10} will be interpreted as being 10mm below the base of the arm, not 10mm below the end effector.
+      This is different from `destination` and `componentName`, where poses are relative to the distal end of a frame.
+  - **Transforms**: These are a list of `PoseInFrame` messages that specify arbitrary other transformations that will be ephemerally added to the frame system at solve time.
+
+- `constraints` ([Constraints](https://pkg.go.dev/go.viam.com/api/service/motion/v1#Constraints)): Pass in optional [motion constraints](./constraints/).
+  By default, motion is unconstrained with the exception of obstacle avoidance.
+
+- `extra` [(map[string]interface{})](https://pkg.go.dev/google.golang.org/protobuf/types/known/structpb): A generic struct, containing extra options to pass to the underlying RPC call.
+
+**Returns:**
+
+- [(bool)](https://pkg.go.dev/builtin#bool): Whether the move was successful.
+- [(error)](https://pkg.go.dev/builtin#error): An error, if one occurred.
+
+For more information, see the [Go SDK Docs](https://pkg.go.dev/go.viam.com/rdk/services/motion#Service).
+
+**Example usage:**
+
+```go {class="line-numbers linkable-line-numbers"}
+ // Access the Motion Service
+motionService, err := motion.FromRobot(robot, "builtin")
+if err != nil {
+  logger.Fatal(err)
+}
+
+// Assumes a gripper configured with name "my_gripper" on the robot
+myFrame := "my_gripper_offset"
+
+goalPose := PoseInFrame(0, 0, 300, 0, 0, 1, 0)
+
+// Move the gripper
+moved, err := motionService.Move(context.TODO(), goalPose, worldState, nil, nil)
+```
+
+{{% /tab %}}
+{{< /tabs >}}
+
+### MoveSingleComponent
+
+*`MoveSingleComponent` allows you to bypass Viam’s internal motion planning entirely, if desired, for a single component.*
```

Comment:
It is italicized for emphasis. Maybe it should be a note box? But that doesn't feel quite right. It is important that it bypasses motion planning because it's part of the motion API so someone might reasonably expect that it employs, rather than bypasses, motion planning.